tide-core/Cargo.toml

```diff
@@ -11,22 +11,13 @@ license = "MIT OR Apache-2.0"
 repository = "https://github.com/rustasync/tide"
 
 [dependencies]
-fnv = "1.0.6"
 futures-preview = "0.3.0-alpha.16"
 http = "0.1"
 http-service = "0.2.0"
-route-recognizer = "0.1.12"
 serde = "1.0.91"
 serde_json = "1.0.39"
-
-[dependencies.http-service-hyper]
-optional = true
-version = "0.2.0"
+route-recognizer = "0.1.12"
 
 [dev-dependencies]
 tide = { path = "../" }
-serde_derive = "1.0.91"
-
-[features]
-default = ["hyper"]
-hyper = ["http-service-hyper"]
+serde_derive = "1.0.91"
```

Comment:
Minor tidbit, but should have a newline at EOF

Reply:
ha, will do!